e2e/tests/workspace/workspace-create.spec.ts

```diff
@@ -1,13 +1,12 @@
 import Link from 'app/element/link';
 import DataPage from 'app/page/data-page';
 import WorkspacesPage, {FIELD} from 'app/page/workspaces-page';
-import {signIn} from 'utils/app-utils';
+import {signIn, performActions} from 'utils/test-utils';
 import Button from 'app/element/button';
 import * as testData from 'resources/data/workspace-data';
-import {performActions} from 'utils/test-utils';
 import {makeWorkspaceName} from 'utils/str-utils';
 
-describe('Creating new workspaces', () => {
+describe.skip('Creating new workspaces', () => {
```

Comment:
un-skip before merge.

Reply:
what do you mean?

Reply:
I skipped clone and create workspaces tests for the PR. I will undo skip first then merge to master.